source/standard_scores.Rmd

```diff
@@ -0,0 +1,1523 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all,-language_info
+    split_at_heading: true
+    text_representation:
+      extension: .Rmd
+      format_name: rmarkdown
+      format_version: '1.2'
+      jupytext_version: 1.14.6
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+```{r setup, include=FALSE}
+source("_common.R")
+```
+
+# Ranks, quantiles and Standard Scores
+
+Sometimes have a set of measures, in some particular units, and we want some
+way to see quickly how these measures compare to one another, and how they may
+compare to other measures, in different units.
+
+Ranks are one way of having an implicit comparison between values.[^ordering]
+Is the value large in terms of the other values (with high rank) — or is it
+small (low rank)?
+
+[^ordering]: To get ranks we have to be able to sort our values in some
+    reasonable way, from high to low or low to high.  This usually makes sense
+    for measured data, where the measurement gives a number, but may not make
+    sense for other data, such as names or labels.  For example, it may ore may
+    not be meaningful to rank names alphabetically.  As usual, this is a matter
+    of judgment.
+
+If We can derive quantile positions from ranks.  These are values from 0 through
+1 that are closer to 1 for high rank values, and closer to 0 for low rank
+values.  Each value in the data has a rank, and a corresponding quantile
+position.  We can also look at the *value* corresponding to each quantile
+position, and these are the *quantiles*. You will see what we mean later in the
+chapter.
+
+Ranks and quantile positions give an idea whether the measure is high or low
+compared to the other values, but they do not immediately tell us whether the
+measure is exceptional or unusual.  To do that, we may want to ask whether the
+measure falls outside the typical range of values — that is, how the measure
+compares to the *distribution* of values.  One common way of doing this is to
+re-express the measures (values) as *standard scores*, where the standard score
+for a particular value tells you how far the value is from the *center* of the
+distribution, in terms of the typical *spread* of the distribution. Standard
```

Comment:
"typical spread" is a bit confusing here, but reading on to see if it is explained later